api/tests/mocha/services/export/feedback-mapper.js

```diff
@@ -10,13 +10,20 @@ describe('Feedback mapper', () => {
   describe('getDocIds', () => {
     it('queries task-messages and returns message ids', () => {
       const options = { some: 'option' };
-
-      sinon.stub(db.medic, 'query').resolves({
+      const queryOptions = {
+        descending: true,
+        endkey: 'feedback-',
+        include_docs: false,
+        startkey: 'feedback-￰'     
```

Comment:
Shouldn't this have `-\ufff0` as the suffix?

As the test is passing it suggests to me that it's not working quite how I expect. Try changing the startkey to "TOTALLYWRONG" and if it still passes you know the assertion isn't right somehow.

My guess is it's because the `options` object you're passing in is being updated by the service, so by the time you make the assertion, `options` already has all the properties you want and the Object.assign does nothing, so you're essentially testing: `chai.expect(options).to.deep.equal(options);`

One fix is to replace the assertion with...

```js
chai.expect(allDocs.args[0]).to.deep.equal([ queryOptions ]);
```

Then update queryOptions to...

```js
const queryOptions = {
  some: 'option',
  descending: true,
  ...
}
```

That way you're testing two completely separate objects so the service updating one won't affect the other.